### PR TITLE
chore: redirect for run biosimulations to point to biosimulations and GUI updates

### PR DESCRIPTION
**Pull request by AlexPatrie**
_What does this PR do?:_
- Adds a redirect for `https://run.biosimulations.dev` to point instead to `https://biosimulations.dev`
- Adds the same aforementioned redirect for "production" (`.org`).
- Adds default values to form directive configurations in `create-project.component` within `libs/shared`.
- Adds default values for x and y axis in 2d line plot creation within `libs/simulation-runs/ui/src/lib/design-line-2d-viz/design-line-2d-viz.component` as well as minor UI (style) refactors, removing unused data headers.
- Minor UI (style) refactors as needed.

### CODE REVIEW

File: apps/platform/src/app/app.component.ts [hunk 1]
```diff
@@ -24,6 +25,7 @@ export class AppComponent implements OnInit, AfterViewInit {
 
   public constructor(
     public config: ConfigService,
+    public dialog: MatDialog,
```
Comment (jcschaff) [inline apps/platform/src/app/app.component.ts]: doesn't seem to be used, am I missing something?

File: libs/analytics/angular-analytics/src/lib/cookie-consent/cookie-consent.component.ts [hunk 2]
```diff
@@ -20,10 +20,10 @@ import {
 })
 export class CookieConsentComponent {
   public cookies: Cookie[] = [
-    new RequiredCookie(),
+    // new RequiredCookie(),
```
Comment (jcschaff) [inline libs/analytics/angular-analytics/src/lib/cookie-consent/cookie-consent.component.ts]: are you sure we should comment out tracking and required.  our advisors suggested that we need to track usage ... does this remove this ability in the future ... maybe we should keep it for now.

File: libs/simulation-project-utils/simulation-project-utils/src/lib/ui/create-project/form-steps/uniform-time-course-simulation.component.ts [hunk 3]
```diff
@@ -44,10 +45,11 @@ export class UniformTimeCourseSimulationComponent implements IFormStepComponent,
   }
 
   public populateFormFromFormStepData(formStepData: FormStepData): void {
-    this.formGroup.controls.initialTime.setValue(formStepData.initialTime);
-    this.formGroup.controls.outputStartTime.setValue(formStepData.outputStartTime);
-    this.formGroup.controls.outputEndTime.setValue(formStepData.outputEndTime);
-    this.formGroup.controls.numberOfSteps.setValue(formStepData.numberOfSteps);
+    // This method is called onInit and thus should set the default vals
```
Comment (jcschaff) [inline libs/simulation-project-utils/simulation-project-utils/src/lib/ui/create-project/form-steps/uniform-time-course-simulation.component.ts:48]: why is this needed if the default values are set in the contructor?  will this play nicely if the user wants to override the defaults?

File: libs/simulation-project-utils/simulation-project-utils/src/lib/ui/create-project/form-steps/upload-model-component.spec.ts [hunk 4]
```diff
@@ -45,7 +45,7 @@ describe('UploadModelComponent', () => {
     fixture.detectChanges();
 
     testFormats = [
-      createTestOntologyTerm('test format 1'),
+      createTestOntologyTerm('test format_2585'),
```
Comment (jcschaff) [inline libs/simulation-project-utils/simulation-project-utils/src/lib/ui/create-project/form-steps/upload-model-component.spec.ts]: is this intentional or a debugging edit?

**Review by jcschaff — CHANGES_REQUESTED** [review]
please take a look at the comments

File: apps/platform/src/app/app.component.ts (hunk 1 above)
Comment (AlexPatrie) [reply apps/platform/src/app/app.component.ts]: This was a debugging artifact which I have now removed.

File: libs/analytics/angular-analytics/src/lib/cookie-consent/cookie-consent.component.ts (hunk 2 above)
Comment (AlexPatrie) [reply libs/analytics/angular-analytics/src/lib/cookie-consent/cookie-consent.component.ts]: Another debugging artifact that was not originally reverted. Now it is uncommented!

File: libs/simulation-project-utils/simulation-project-utils/src/lib/ui/create-project/form-steps/uniform-time-course-simulation.component.ts (hunk 3 above)
Comment (AlexPatrie) [reply libs/simulation-project-utils/simulation-project-utils/src/lib/ui/create-project/form-steps/uniform-time-course-simulation.component.ts:48]: Great question. In angular forms, there is a difference between setting a default value (that which is specified in `constructor`) and setting a form value such that it actually renders. The default value specified in this component's constructor is only collected `onSubmit()` (called by the parent component), and thus will not render on the screen as a `Change` in the form data. This call to `.setValue()` originally set the data from the `formStepData` of the previous step, which was `null`. The form data source is actively subscribed to user changes, so this explicit setting will not mess with new override specifications.

TLDR; Explicitly setting these values is required for rendering the default values and will not tamper with user specifications.

File: libs/simulation-project-utils/simulation-project-utils/src/lib/ui/create-project/form-steps/upload-model-component.spec.ts (hunk 4 above)
Comment (AlexPatrie) [reply libs/simulation-project-utils/simulation-project-utils/src/lib/ui/create-project/form-steps/upload-model-component.spec.ts]: This is another matter relating to default values in angular. `format_2585` is the tag for the non-spatial continuous framework option which we discussed should be the default value for the modeling framework option in `create-project/upload-model-component`. The format of this test requires this value.

File: libs/simulation-project-utils/simulation-project-utils/src/lib/ui/create-project/form-steps/uniform-time-course-simulation.component.ts (hunk 3 above)
Comment (jcschaff) [reply libs/simulation-project-utils/simulation-project-utils/src/lib/ui/create-project/form-steps/uniform-time-course-simulation.component.ts:48]: thanks, great explanation.

File: libs/simulation-project-utils/simulation-project-utils/src/lib/ui/create-project/form-steps/upload-model-component.spec.ts (hunk 4 above)
Comment (jcschaff) [reply libs/simulation-project-utils/simulation-project-utils/src/lib/ui/create-project/form-steps/upload-model-component.spec.ts]: okay thanks.

**Review by jcschaff — APPROVED** [review]
Thanks for answering my questions, approved!
